certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -271,14 +316,20 @@ SETUP
  USEROPTIONSCHR(4)         = "Renew certificate"
  USEROPTIONSCHR(5)         = "Disburse funds by check"
  USEROPTIONSCHR(6)         = "Suspend certificate"
- DEBUGMODE                 = FALSE
  SYMXINSTANCE              = @SYMXINSTANCEID
  SYMXCLIENTNUM             = @CLIENTNUMBER
 
- FOR TMPLOOP=1 TO INDENTMAX
+
+ FOR BNOLOCALLOOP=1 TO BNOINDENTMAX
   DO
-   INDENT(TMPLOOP)=REPEATCHR(" ",TMPLOOP*2)
+   BNOINDENT(BNOLOCALLOOP)=REPEATCHR(" ",BNOLOCALLOOP)
   END
+ BNOIDLENGTH=GETDATANUMBER(GETPARAMIDLENGTH)
+ BNOSLTYPELENGTH=GETDATANUMBER(GETPARAMTYPELENGTH)
+ BNOSLTYPESMAX=VALUE(SEGMENT("9999",1,BNOSLTYPELENGTH))
+
+ BNOTRUEFALSECHR(TRUE)="true"
+ BNOTRUEFALSECHR(FALSE)="false"
 END [SETUP]
 
 PRINT TITLE="BANNO.CDRENEW.V1.POW"
```

Comment:
I can't leave a comment there, but I noticed on line 342 and 343 it does not allow running the program in memomode, which is fine:
 IF ERRORCODE=0 AND
    SYSTEMMEMOMODE=FALSE THEN

But just above that on line 339 and 340 you have the following two lines with a Variable called ALLOWINMEMOMODE: 
IF ALLOWINMEMOMODE=FALSE THEN
  CALL VALIDATESYSMEMOMODE

Do you have plans to allow CU's to set ALLOWINMEMOMODE to true and then let them use CD Renew in memo Mode?

Reply:
There was some discussion at the time about allowing/not allowing the program to run in memo mode - and in some cases in the past, we have allowed the CU to set that value in the CFG Letter file. In this case, as the code hasn't changed in this update, I'm not seeing any benefit to change it. Also, by leaving it in, we can, more easily, add that option in the future.